### PR TITLE
My review

### PR DESCRIPTION
**Pull request by george827**
### Survey form

- Review of the previous project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,21 +14,28 @@
...
     Please read the README below in the JS Editor before beginning.
+
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
+
     - The freeCodeCamp Team
 
+
     -->
```
Comment (george827) [inline index.html:27]: kindly remove unmeaningful comments to your code.  The comments need to be relevant to the code

File: index.html [hunk 2]
```diff
@@ -37,31 +44,40 @@ <h1 id="title"> freecode survay </h1>
...
+
     <textarea id= "survay-form" cols="40" rows = "5"> comment here</textarea>
     <button id= "submit" type="submit"> sumbmit </button> 
 
 
+    
+
        </form>
   </body>
   <P>Thanks for help</P>
```
Comment (george827) [inline index.html:83]: kindly ensure you use proper HTML semantics, the p  tag is supposed to be within the HTML body

File: index.html [hunk 3]
```diff
@@ -37,31 +44,40 @@ <h1 id="title"> freecode survay </h1>
...
     <label for ="gender"> Option five <input value="one" type="checkbox" name="name"> <br>
     <label for ="gender"> Option six <input value="one" type="checkbox" name="name"> <br>
+
     <textarea id= "survay-form" cols="40" rows = "5"> comment here</textarea>
     <button id= "submit" type="submit"> sumbmit </button> 
 
 
+    
+
        </form>
```
Comment (george827) [inline index.html:81]: Kindly remove the unused spaces in your code. This will make your work look clean

File: index.html [hunk 4]
```diff
@@ -37,31 +44,40 @@ <h1 id="title"> freecode survay </h1>
...
+
 <option> first choice </option>
 <option> second choice </option>
 <option> third choice </option>
 <option> forth choice </option>
-<option> fifth choice </option>
-<option> sixth choice </option>
+
+<!-- <option> fifth choice </option> -->
+<!-- <option> sixth choice </option> -->
```
Comment (george827) [inline index.html:56]: kindly ensure you don't have unused code as comments. Unused code makes your work not clean

**Review by george827 — COMMENTED** [review]
Good work so far
All linters are passing ✔️
Correct GitHub workflow ✔️
Meaningful commit messages ✔️

**Review by ichala — APPROVED** [review]
Hi @george827 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Great use of GitHub tools ✔️ 
- You explained why you requested changes ✔️ 
- Your comments are clear and straightforward ✔️  

Your project is complete!  🎉 

# Optional suggestions :

- [ ] [OPTIONAL] try to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**